climate_emotions_map/make_descriptive_plots.py

```diff
@@ -209,85 +266,164 @@ def make_impact_plot_traces(df: pd.DataFrame):
     )
     data_impact = data_impact.sort_values(COL_DEMOGRAPHIC_VARIABLE)
     traces = []
-    for category in ["No", "Yes"]:
+    for category in ["Yes"]:
         data_category = data_impact.loc[data_impact[COL_CATEGORY] == category]
+        x = data_category[COL_DEMOGRAPHIC_VARIABLE].apply(
+            get_demographic_variable_to_display
+        )
         traces.append(
             go.Bar(
-                x=data_category[COL_DEMOGRAPHIC_VARIABLE].apply(
-                    get_demographic_variable_to_display
+                x=x.apply(lambda x: wrap_text_label(x, width=text_wrap_width)),
+                y=data_category[COL_PERCENTAGE] * 100,
+                # text=[
+                #     f"{percentage*100:.1f}% ({n})"
+                #     for percentage, n in zip(
+                #         data_category[COL_PERCENTAGE], data_category[COL_N]
+                #     )
+                # ],
+                text=x.apply(
+                    lambda x: wrap_text_label(x, width=text_wrap_width)
                 ),
-                y=data_category[COL_N],
-                customdata=data_category[COL_PERCENTAGE] * 100,
+                customdata=list(zip(x, data_category[COL_N])),
                 hovertemplate=(
-                    "<b>%{x}</b>"
-                    f"<br>{category}: %{{y}} (%{{customdata:.1f}}%)"
+                    "<b>%{customdata[0]}</b>"
+                    ": %{y:.2f}% (%{customdata[1]})"
                     "<extra></extra>"
                 ),
+                marker_color=marker_color,
+                hoverinfo="none",
                 name=category,
             )
         )
     return traces
 
 
-def make_descriptive_plots(state: str | None = None) -> go.Figure:
+def make_descriptive_plots(
+    state: str | None = None, margins=None, text_wrap_width=14
+) -> go.Figure:
+
+    if margins is None:
+        margins = {"l": 0, "r": 0, "t": 20, "b": 0}
 
     # get data to plot
     if state is None:
         data = SAMPLEDESC_WHOLESAMPLE
     else:
         data = SAMPLEDESC_STATE.loc[SAMPLEDESC_STATE["state"] == state]
 
+    row_heights = [
+        n_categories + 0.5
```

Comment:
Related to other comment, I think you may want to have this be a product or division w/ something related to the vertical spacing